tests/src/com/unciv/testing/TranslationTests.kt

```diff
@@ -188,4 +185,48 @@ class TranslationTests {
             allWordsTranslatedCorrectly
         )
     }
+    
+    @Test
+    fun wordBoundaryTranslationIsFormattedCorrectly() {
+        val translationEntry = translations["\" \""]!!
+                
+        var allTranslationsCheckedOut = true
+        for ((language, translation) in translationEntry) {
+            if (!translation.startsWith("\"")
+                || !translation.endsWith("\"")
+                || translation.count { it == '\"' } != 2
+            ) {
+                allTranslationsCheckedOut = false
+                println("Translation of the word boundary in $language was incorrectly formatted")
+            }
+        }
+        
+        Assert.assertTrue(
+            "This test will only pass when the word boundrary translation succeeds",
+            allTranslationsCheckedOut
+        )
+    }
+    
+    @Test
+    fun allConditionalsAreContainedInConditionalOrderTranslation() {
```

Comment:
Not sure whether relaxing this might be better. How often did we explain test failures to translators recently? You did the code to default to original ordering...

Reply:
I don't have a particularly strong reasoning either way, but more relaxation does mean more possibility for unintended typo's, and translators than need to fix that anyway for it to work, regardless of how whether we have to explain that. At least this way we know that they made a mistake, instead of them being confused why three versions later it isn't working as they expected it to